exchange.htm

```diff
@@ -7,28 +7,29 @@
         <div class="row">
             <div class="col-12 col-sm-12 col-md-12 col-lg-12">
                 <h3>How to buy Gridcoin?</h3>
-                <p>On most centralized exchanges you need to first <a href="https://www.bitcoin.com/buy-bitcoin">buy Bitcoin</a> in order to buy Gridcoin.</p>
+                <p>On most exchanges you need to first <a href="https://www.bitcoin.com/buy-bitcoin">buy Bitcoin</a> in order to buy Gridcoin.</p>
             </div>
         </div>
-
+        <!--
         <div class="row text-center">
```

Comment:
I think you've commented out an extra div here` <div class="row text-center">` as the closing pair for this div is not in the commented section.

Reply:
The div was extraneous.  It showed as red in my editor and there is no visible change with it removed